### PR TITLE
Help text

### PR DESCRIPTION
**Pull request by yetiforce-wojtek77**
## Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code is written according to the guidelines found [here] (https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

**Comment by yetiforce-wojtek77** [comment]
image

**Comment by yetiforce-wojtek77** [comment]
image

**Comment by yetiforce-wojtek77** [comment]
image

File: layouts/basic/modules/Settings/ConfReport/Index.tpl [hunk 1]
```diff
@@ -82,7 +82,7 @@
 								<tr {if $item.status}class="danger"{/if}>
 									<td>
 										<label>{$key}</label>
-										{if isset($item.help) && $item.status}<a href="#" class="popoverTooltip pull-right" data-trigger="focus" data-placement="rigth" data-content="{App\Language::translate($item.help, $MODULE)}"><i class="glyphicon glyphicon-info-sign"></i></a>{/if}
+										{if isset($item.help) && $item.status}<a href="#" class="popoverTooltip pull-right" data-trigger="focus" data-placement="rigth" data-content="{\App\Purifier::encodeHtml(App\Language::translate($item.help, $MODULE))}"><i class="glyphicon glyphicon-info-sign"></i></a>{/if}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Settings/ConfReport/Index.tpl]: App\Purifier::encodeHtm ?

File: layouts/basic/modules/Settings/ConfReport/Index.tpl [hunk 2]
```diff
@@ -116,7 +116,7 @@
 								<tr {if $item.status}class="danger"{/if}>
 									<td>
 										<label>{$key}</label>
-										{if isset($item.help) && $item.status}<a href="#" class="popoverTooltip pull-right" data-trigger="focus" data-placement="rigth" data-content="{App\Language::translate($item.help, $MODULE)}"><i class="glyphicon glyphicon-info-sign"></i></a>{/if}
+										{if isset($item.help) && $item.status}<a href="#" class="popoverTooltip pull-right" data-trigger="focus" data-placement="rigth" data-content="{\App\Purifier::encodeHtml(\App\Language::translate($item.help, $MODULE))}"><i class="glyphicon glyphicon-info-sign"></i></a>{/if}
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Settings/ConfReport/Index.tpl]: App\Purifier::encodeHtm ?

File: languages/de_de/Settings/ConfReport.php [hunk 3]
```diff
@@ -78,6 +78,18 @@
 	'HTACCESS_HELP_TEXT' => '.htaccess Dateien können verwendet werden um die Servereinstellungen anzupassen. YetiForceCRM benötigt den Zugriff auf die Dateien um korrekt zu funktionieren.',
 	'PUBLIC_HTML_HELP_TEXT' => 'Wir empfehlen das System so zu installieren, dass nur das öffentliche HTML Verzeichnis benutzt wird. Es erhöht die Datensicherheit, wenn auf nichtöffentliche Verzeichnisse und Dateien nicht direkt über die Adresse(URL) zugegriffen werden kann.',
 	'SESSION_REGENERATE_HELP_TEXT' => 'Wir empfehlen die Erneuerung von Session IDs zu aktivieren, da diese Option es schwerer macht die Benutzersitzung zu übernehmen. Änderungen sollten in der Hauptkonfigurationsdatei durchgeführt werden.',
+	'SESSION_USE_STRICT_MODE_HELP_TEXT' => 'Determines if the server rejects an uninitialized session. Due to security concerns we recommend enabling this option.',
```
Comment (mariuszkrzaczkowski) [inline languages/de_de/Settings/ConfReport.php]: SESSION_USE_STRICT_MODE_HELP_TEXT  >> LBL_SESSION_USE_STRICT_MODE_HELP_TEXT

File: layouts/basic/modules/Settings/ConfReport/Index.tpl (hunk 1 above)
Comment (yetiforce-wojtek77) [reply layouts/basic/modules/Settings/ConfReport/Index.tpl]: because of special chars

**Pull request merged**